dask/dataframe/tests/test_dataframe.py

```diff
@@ -2535,13 +2535,6 @@ def test_drop_columns(columns):
     assert_eq(df.drop(columns=columns), ddf2)
 
 
-def test_gh580():
-    df = pd.DataFrame({"x": np.arange(10, dtype=float)})
-    ddf = dd.from_pandas(df, 2)
-    assert_eq(np.cos(df["x"]), np.cos(ddf["x"]))
-    assert_eq(np.cos(df["x"]), np.cos(ddf["x"]))
```

Comment:
Why remove this?

Reply:
I started out trying to just add a comment documenting what this test was doing, since it's a bit cryptic, and ultimately decided it was stale/redundant enough to be considered "dead code".

Looking into the history (#580, #582) it seems it was verifying some workaround related to ufuncs that has since been superseded with more correct behavior. I'm not sure if the duplicated assertion is intentional or was ever significant, and I believe e.g. `test_ufunc.py` covers whatever is being tested here (and then some).

Happy to leave it in, but in that case maybe someone else should try their hand at documenting exactly what this test is doing? If no one can, that seems like a good motivation to remove it, I think?